python/social_bot/envs/grocery_ground.py

```diff
@@ -0,0 +1,232 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+"""
+A simple enviroment for an agent play on a groceryground
+"""
+import os
+import logging
+import numpy as np
+import math
+import random
+import PIL
+
+import gym
+from gym import spaces
+from collections import OrderedDict
+
+import social_bot
+from social_bot import teacher
+from social_bot.teacher import TeacherAction
+from social_bot.envs import simple_navigation
+from social_bot.envs.simple_navigation import GoalTask
+from social_bot.envs.simple_navigation import DiscreteSequence
+import social_bot.pygazebo as gazebo
+
+logger = logging.getLogger(__name__)
+
+
+class GroceryGroundGoalTask(GoalTask):
```

Comment:
The functionality of this class is almost same as GoalTask. We should update GoalTask and use GoalTask instead. And GoalTask should be moved a separate file as it's not used only by SimpleNavigation

Reply:
Yes, we should.

Reply:
It seems to me that we can add one more argument `goal_random_range" to GoalTask so that we can reuse it instead of implementing a new one.